src/components/Layout.js

```diff
@@ -74,6 +74,7 @@ const Layout = ({ children, withSidebar }) => {
         <Header
           siteTitle={data.site.siteMetadata.title}
           scrolled={headerScrolled}
+          size={'large'}
```

Comment:
Since this is the only place `<Header />` is used, and since `<Search />` is only used inside `<Header />`, I think we should just get rid of this prop all the way down and assume that's the size.